internal/testutil/adaptermocks/objectsetphase_mock.go

```diff
@@ -0,0 +1,42 @@
+package adaptermocks
```

Comment:
I think it would be better to move this to `internal/testutil/controllersmocks/objectsetphasereconciler_mock.go`.

Reply:
Fixed.